### PR TITLE
[#9] Add darren to design

### PR DESCRIPTION
**Pull request by xhr15**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -162,6 +165,7 @@
             For details on what kinds of behaviour are not tolerated and consequences for violating these rules, we refer to the <a href="http://rubyberlin.github.io/code-of-conduct">Berlin Code of Conduct</a>.
           </p>
         </div>
+
```
Comment (planktonic) [inline index.html:167]: Nitpick: was this deliberate?

**Review by planktonic — APPROVED** [review]
Yup, looks good

File: index.html (hunk 1 above)
Comment (xhr15) [reply index.html:167]: the newline? Yes but if that is ugly I will delete it

Comment (xhr15) [reply index.html:167]: ok, had a look again, deleted it.

Comment (planktonic) [reply index.html:167]: ok! Not ugly, just inconsistent with the rest of the file :)

**Pull request merged**